### PR TITLE
fix(README): Change instructions for installing badger in README.md

### PR DESCRIPTION
**Pull request by vmrajas**
It was found out that the current ```go get ``` instruction to install badger was not working properly.
This PR changes the installation instruction.

Related Discuss issue: https://discuss.dgraph.io/t/how-to-use-cli-badger-tool/9729

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/badger/1495)

### CODE REVIEW

**Review by jarifibrahim — CHANGES_REQUESTED** [review]
Please make the same change in the `docs/` directory as well so that it reflects on https://dgraph.io/docs/badger/get-started/#installing page.

> *[Reviewable](https://reviewable.io/reviews/dgraph-io/badger/1495#-:-MG8bR_87sUAMUJlT-pk:b-nxtosy)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @ashish-goswami, @jarifibrahim, @manishrjain, and @vmrajas)

___
*[README.md, line 69 at r1](https://reviewable.io/reviews/dgraph-io/badger/1495#-MG8bg-x7KIKb7m0YhKS:-MG8bg-x7KIKb7m0YhKT:b-4yo3i9) ([raw file](https://github.com/dgraph-io/badger/blob/5e2b3a71e6b4f1035ddf92f1a564297f1fe31f76/README.md#L69)):*
> ```Markdown
> ## Getting Started
> 
> ### Installing v2.0.0
> ```

Why do we want users to install this specific version? The latest release is v2.2007.1 and go get would fetch it by default.

**Review by vmrajas — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/badger/1495)* status: 0 of 3 files reviewed, 1 unresolved discussion (waiting on @ashish-goswami, @jarifibrahim, and @manishrjain)

___
*[README.md, line 69 at r1](https://reviewable.io/reviews/dgraph-io/badger/1495#-MG8bg-x7KIKb7m0YhKS:-MG8tCmIBmuy9ylikWC6:b-896fix) ([raw file](https://github.com/dgraph-io/badger/blob/5e2b3a71e6b4f1035ddf92f1a564297f1fe31f76/README.md#L69)):*
<details><summary><i>Previously, jarifibrahim (Ibrahim Jarif) wrote…</i></summary><blockquote>

Why do we want users to install this specific version? The latest release is v2.2007.1 and go get would fetch it by default. 
</blockquote></details>

Done.

File: docs/content/get-started/index.md [hunk 1]
```diff
@@ -5,19 +5,24 @@ aliases = ["/get-started"]
 
 
 ## Installing
-To start using Badger, install Go 1.12 or above and run `go get`:
+To start using Badger, install Go 1.12 or above. Badger v2 needs go modules and run `go get`:
```
Comment (jarifibrahim) [inline docs/content/get-started/index.md:8]:
```suggestion
To start using Badger, install Go 1.12 or above. Badger v2 needs go modules. Run the following command to retrieve the library.
```

**Review by jarifibrahim — APPROVED** [review]
Looks good to me. I've got one comment. 
I see that the image on https://dgraph.io/docs/badger is missing. @vmrajas if you can fix it in this PR, that would be great.

**Review by vmrajas — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/badger/1495)* status: 0 of 4 files reviewed, 2 unresolved discussions (waiting on @ashish-goswami, @jarifibrahim, and @manishrjain)

___
*[docs/content/get-started/index.md, line 8 at r2](https://reviewable.io/reviews/dgraph-io/badger/1495#-MG8t-oNCMdN2vr7Cgob-r2-8:-MGcJpfQ7ZKRtp_hTbGM:b-896fix) ([raw file](https://github.com/dgraph-io/badger/blob/e98e9f8d7113bc2bafc0274d170983476c7b9bf9/docs/content/get-started/index.md#L8)):*
<details><summary><i>Previously, jarifibrahim (Ibrahim Jarif) wrote…</i></summary><blockquote>

```suggestion
To start using Badger, install Go 1.12 or above. Badger v2 needs go modules. Run the following command to retrieve the library.
```
</blockquote></details>

Done.

**Pull request merged**